applicationconfig/client/README.md

```diff
@@ -0,0 +1,188 @@
+# Azure App Configuration client library for Java
+Azure App Configuration is a managed service that helps developers centralize their application configurations simply and securely.
+
+Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to store all the settings for your application and secure their accesses in one place.
+
+Use the client library for App Configuration to create and manage application configuration settings.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][azconfig_docs]
```

Comment:
File an issue here to fix these links when they are known.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/3437